content/js2/blocks/arrays/index.md

```diff
@@ -27,7 +27,7 @@ We can declare an **array literal** as follows:
 const items = [4.6, 5.03, 7.99, 8.01];
 ```
 
-Notice the identifier for the array is **items**. The word `items` is plural as arrays can store multiple pieces of information.
+Notice the identifier for the array is **items**. We chose to use the plural word `items` instead of the singular `item`, because arrays can store multiple pieces of information.
```

Comment:
I want to make it clear this is a "programmers making life easier for programmers" decision, not a "arrays must have plural names for the language" decision.